eager2-core/Cargo.toml

```diff
@@ -19,4 +19,6 @@ proc-macro2 = { version = "1", optional = true }
 quote = { version = "1", optional = true }
 
 [features]
-"testing" = ["proc-macro2", "quote"]
+testing = ["dep:proc-macro2", "dep:quote"]
+proc-macro-support = ["dep:proc-macro2", "dep:quote"]
```

Comment:
I'd like to avoid any dependencies in `eager-core` in order to facilitate fast compiles. It's why I've chosen to avoid `syn` and why I only bring in these dependencies for testing.